server/test/util/body-parser.js

```diff
@@ -67,6 +66,12 @@ test('convertWpVideo', t => {
   );
 });
 
+test('convertTweet', t => {
+  const bodyParts = bodyParser(wpApiResp.content);
+  const tweets = bodyParts.filter(part => part.type === 'tweet');
+  t.is(tweets.length, 1);
```

Comment:
Yes – I added one to the end of the `content` (if you load the diff it's at the end of the block of green).